### PR TITLE
streammusic 1.1.10 (new cask)

### PR DESCRIPTION
**Pull request by jibwf**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by razvanazamfirei** [comment]
Thank you for your submission to homebrew-cask @jibwf. Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) and a [livecheck](https://docs.brew.sh/Cask-Cookbook#stanza-livecheck) to this cask.

**Comment by jibwf** [comment]
> Thank you for your submission to homebrew-cask @jibwf. Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) and a [livecheck](https://docs.brew.sh/Cask-Cookbook#stanza-livecheck) to this cask.
> 
> Also, please double check the cask name follows our [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).

Done.

File: Casks/s/stream-music.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask "stream-music" do
```
Comment (reitermarkus) [inline Casks/s/stream-music.rb:1]: Should be `streammusic` since there is no hyphen in the app name.

File: Casks/s/stream-music.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  version "1.1.10"
+  sha256 "fc1d3a12d2463f5bacbbb6936d81bb5e97b394cd545ed349aa079272fe561bc5"
+
+  url "https://oss.aqzscn.cn/stream-music/versions/#{version}/%E9%9F%B3%E6%B5%81v#{version}.dmg"
+  name "StreamMusic"
+  desc "Music client compatibility with self-hosted music services"
```
Comment (reitermarkus) [inline Casks/s/stream-music.rb:7]:
```suggestion
  desc "Music client compatible with self-hosted music services"
```

**Review by razvanazamfirei — APPROVED** [review]

**Comment by razvanazamfirei** [comment]
Thanks, @jibwf!

**Pull request merged**